Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1378,7 +1378,7 @@ extension BrowserViewController: BrowserToolbarDelegate {
                 if isBookmarked {
                     self.removeBookmark(url)
                 } else {
-                    self.addBookmark(url, title: tab.title)
+                    self.addBookmark(url, title: self.title)
                 }
```

Comment:
I made this change because things didn't compile without it, but I think I committed it accidentally.